services/pushover/service.go

```diff
@@ -75,6 +75,7 @@ func (s *Service) Test(options interface{}) error {
 	if !ok {
 		return fmt.Errorf("unexpected options type %t", options)
 	}
+	var ts *time.Time
```

Comment:
Just removed the logic around timestamps. It didn't seem particularly useful.

The pointer was a remanent from when I thought their API accepted JSON so that when I tagged it with `omitempty` it wouldn't appear in the JSON I sent.